rainfall/style.css

```diff
@@ -1,25 +1,38 @@
 /*
 Theme Name: Rainfall
-Theme URI: TBD
+Theme URI: https://wordpress.com/theme/rainfall
 Author: Automattic
 Author URI: https://automattic.com
-Description: TBD
-Requires at least: 5.8
-Tested up to: 5.9
+Description: Rainfall is a clean, objective blogging theme strongly inspired by Swiss Design. Its minimalist functionality is balanced by a strong accent color, beautiful photography and post templates with sidebars.
+Requires at least: 6.0
+Tested up to: 6.0
 Requires PHP: 5.7
 Version: 0.0.7
 License: GNU General Public License v2 or later
-License URI: https://raw.githubusercontent.com/Automattic/themes/trunk/LICENSE
-Template:
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
 Text Domain: rainfall
-Tags: one-column, custom-colors, custom-menu, custom-logo, editor-style, featured-images, full-site-editing, rtl-language-support, theme-options, threaded-comments, translation-ready, wide-blocks
+Tags: two-columns, custom-colors, custom-menu, custom-logo, editor-style, featured-images, full-site-editing, block-patterns, rtl-language-support, theme-options, threaded-comments, translation-ready, auto-loading-homepage, blog-homepage
 */
 
+/*
+ * Font smoothing
+ */
+body {
+	-moz-osx-font-smoothing: grayscale;
```

Comment:
I don't think we should be doing this.

Reply:
Yeah I'm not sure about this either! I feel like it's only there to improve font appearance for a small number of users.

Reply:
I removed it